includes/optional-modules/class-rss.php

```diff
@@ -244,14 +261,31 @@ public static function render_url_metabox( $feed_post ) {
 			return;
 		}
 
-		$feed_url = self::get_feed_url( $feed_post );
-
+		$rss_feed_url  = self::get_feed_url( $feed_post );
+		$atom_feed_url = self::get_feed_url( $feed_post, 'atom' );
 		?>
-		<h3>
-			<a href='<?php echo esc_url( $feed_url ); ?>' target='_blank'>
-				<?php echo esc_url( $feed_url ); ?>
-			</a>
-		</h3>
+		<table>
+			<tr>
+				<td><h3><?php esc_html_e( 'RSS :-', 'newspack-plugin' ); ?></h3></td>
+				<td>
+					<h3>
+						<a href="<?php echo esc_url( $rss_feed_url ); ?>" target="_blank">
+							<?php echo esc_url( $rss_feed_url ); ?>
+						</a>
+					</h3>
+				</td>
+			</tr>
+			<tr>
+				<td><h3><?php esc_html_e( 'Atom :-', 'newspack-plugin' ); ?></h3></td>
```

Comment:
The `:-` looks odd. How about a simple dash (here & in the other instance above)?
 ```suggestion
				<td><h3><?php esc_html_e( 'Atom -', 'newspack-plugin' ); ?></h3></td>
```